hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1830,14 +1833,13 @@
           - homepage
           - id
           - reference_code
+          - submission_email
```

Comment:
I don't think the `public` role should have access to this column? Please double-check existing permissions on current `teams.submission_email` column - it's specifically excluded from public role "select" access!

Reply:
Ah sorry! Good catch, I'll update now